tests/sentry/tsdb/test_redis.py

```diff
@@ -70,6 +71,9 @@ def test_get_model_key(self):
         result = self.db.get_model_key("foo")
         assert result == "bf4e529197e56a48ae2737505b9736e4"
 
+        result = self.db.get_model_key("我爱啤酒")
+        assert result == "26f980fbe1e8a9d3a0123d2049f95f28"
```

Comment:
Just noting that this hash works in python 2 before this change as well, so it will be consistent for any existing unicode strings. After auditing this, I actually think we never pass any unicode characters to this method, but better to be safe.